### DISCUSSION TITLE
Segement type envvar has gone... and now?

### DISCUSSION THREAD

**Discussion by mrolli** [JanDeDobbeleer/oh-my-posh#1686] [Troubleshoot] [answered]
I used the envvar segment type to add the value of environment variables to my $PS1, for example:

        {
          "foreground": "#458588",
          "properties": {
            "postfix": " ",
            "var_name": "CONDA_PROMPT_MODIFIER"
          },
          "style": "plain",
          "type": "envvar"
        },

Now that envvar type has gone after being deprecated for some time now, what am I supposed now to simply output the value of an environment variable instead? I'm on a (z|ba)sh...

The link on the [command page](https://ohmyposh.dev/docs/command), quote "(look at abusing [environment variables](https://ohmyposh.dev/docs/environment) for that)" is broken. Hoped to find the solution there.

**Comment by JanDeDobbeleer** [comment, marked as answer]
All you need now is a text segment with the following template: `{{ .Env. CONDA_PROMPT_MODIFIER }}`.

Just change the type to text and replace `var_name` to `template` with above value:

```

    {
      "foreground": "#458588",
      "properties": {
        "template": "{{ .Env.CONDA_PROMPT_MODIFIER }}"
      },
      "style": "plain",
      "type": "text"
    },

```

**Reply by mrolli** [reply]
Works like a charm. Thank you.

But there's a little error in your template, which you might want to fix. There's a space between `.Env.` and `CONDA_....`, which in turn produces a "invalid template text" error.

**Reply by JanDeDobbeleer** [reply]
You're right, fat fingers 😂
